### DISCUSSION TITLE
Train test validation for NeuralProphet

### DISCUSSION THREAD

**Discussion by Adi013007** [ourownstory/neural_prophet#1171] [Q&A - get help using NeuralProphet] [answered]
The evaluation dataframe is not collecting evaluation dataframe

```
def run_train_test_eval(dataframe,dataframe_group):
    model = NeuralProphet(**params)
    df_train, df_test = model.split_df(df=dataframe, freq="MS", valid_p=0.4)
    eval_train = model.fit(df=df_train, freq="MS")
    eval_test = model.test(df = df_test)
    future = model.make_future_dataframe(df = dataframe, periods=12)
    forecast = model.predict(future)
    fig = model.plot(forecast)

```
The training and testing set were created properly

The forecast plot is also not showing actual value

Evaluation dataframe
![image](https://user-images.githubusercontent.com/81505276/221421828-54a213a3-08ee-4595-8d4e-58f7f9f01bf4.png)

Forecast plot
![April](https://user-images.githubusercontent.com/81505276/221421880-df204d80-48cc-4baf-8ee4-1eb0ddb82a0a.png)

**Comment by judussoari** [comment, marked as answer]
Will be solved with our new release 0.6.0
